tasks/build.yml

```diff
@@ -1,16 +1,14 @@
 ---
 
-- name: Import the NodeSource GPG key into apt
-  become: yes
-  apt_key:
-    url: https://deb.nodesource.com/gpgkey/nodesource.gpg.key
-    state: present
-
+  # in order to use `apt_key` with nodejs repos you need python >= 2.7.9
+  # with SNI support
 - name: Add NodeSource deb repository
   become: yes
-  apt_repository:
-    repo: "deb https://deb.nodesource.com/node_{{ nodejs.version }}.x {{ ansible_distribution_release }} main"
-    state: present
+  shell: "{{ item }}"
+  with_items:
```

Comment:
I think you should use the validate_certs option of the apt_key and apt_repository modules if possible.